### ISSUE TITLE
only display first 5 licenses 

### ISSUE THREAD

**Issue by JJ-Author** [dbpedia/databus#129]
and then use show remaining x licenses
![image](https://github.com/dbpedia/databus/assets/2630603/02fa5054-1355-4c6c-b995-ebfc8ada1148)

**Comment by Ayush-Kumar91221** [comment]
I would like to take on this issue. Is this issue still open?

**Comment by Ayush-Kumar91221** [comment]
Yes after my [issue 146](https://github.com/dbpedia/databus/issues/146) is resolved, I will get onto it
